File: Estructura de datos/Arrays Buscador/ArraysBuscador.js
function busca(nums, x) {
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] === x) return i;
    }
    return -1;
}

const datos = [5, 2, 8, 1, 9, 3];
const val = 8;

const pos = busca(datos, val);

if (pos !== -1)
    console.log(`Valor ${val} en pos ${pos}`);
else
    console.log(`Valor ${val} no hallado`);

File: Estructura de datos/Arrays Buscador/ArraysBuscador.cpp
#include <iostream>
#include <vector>
using namespace std;

int busca(const vector<int>& v, int x) {
    for (int i = 0; i < v.size(); i++) {
        if (v[i] == x) return i;
    }
    return -1;
}

int main() {
    vector<int> nums = {5, 2, 8, 1, 9, 3};
    int val = 8;

    int pos = busca(nums, val);

    if (pos != -1)
        cout << "Valor " << val << " en pos " << pos << endl;
    else
        cout << "Valor " << val << " no hallado" << endl;

    return 0;
}
